daml-script/test/daml/TestDisclosures.daml

```diff
@@ -0,0 +1,70 @@
+module TestDisclosures where
```

Comment:
we’re not actually running that test anywhere are we?

Reply:
it should be run as part of `script-test-1.dev`

Reply:
yikes. Do we have a proper place to move this kind of test?

Reply:
I think the file is fine but you need to actually write a test that uses it. For the gRPC APi that means enabling it somewhere in https://github.com/digital-asset/daml/blob/main/daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

For Daml Studio, you need to enable it in https://github.com/digital-asset/daml/blob/main/compiler/damlc/tests/src/DA/Test/ScriptService.hs

(yes the fact that those are two separate tests is dumb see https://github.com/digital-asset/daml/issues/8753)